zapisy/apps/enrollment/records/models/opening_times.py

```diff
@@ -55,6 +54,22 @@ def is_after_t0(cls, student: Student, semester: Semester, time: datetime) -> bo
             return False
         return True
 
+    @classmethod
+    def get_ects_ranking(cls) -> Dict[int, int]:
+        """Computes ECTS ranking, the higher the ranking the greater the time bonus.
+
+        The populate_t0 function uses this ranking to calculate
+        the opening time for selected students.
+        """
+        students = Student.get_active_students()
+
+        distinct_ects: Set[int] = set([student.ects for student in students])
```

Comment:
```suggestion
        distinct_ects: Set[int] = set( student.ects for student in students )
```
Jeśli generatory składane są jakieś nieczytelne i konfudujące, to nie będę się upierał przy tej zmianie.

Reply:
Według mnie wersja z nawiasami kwadratowymi jest bardziej czytelna, ale zobaczyłem jak wyglądają podobne takie miejsca w kodzie projektu i dodatkowe nawiasy nie są tam uzywane, więc zastosowałem poprawki.